### PR TITLE
document new unicode deprecation notification for the future 5.15 rel…

### PR DESCRIPTION
**Pull request by nadja-muller**
…ease

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by Lojjs — APPROVED** [review]

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 1]
```diff
@@ -16,6 +16,29 @@ New features are added to the language continuously, and occasionally, some feat
...
+
+a|
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+RETURN 1 as my\u0085identifier
+----
+a|
+The Unicode character \`\u0085` is deprecated for unescaped identifiers and will be considered as a whitespace character in the future. To continue using it, escape the identifier by adding backticks around the identifier.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
The Unicode character \`\u0085` is deprecated for unescaped identifiers and will be considered as a whitespace character in the future. 
To continue using it, escape the identifier by adding backticks around the identifier.
```

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 2]
```diff
@@ -16,6 +16,29 @@ New features are added to the language continuously, and occasionally, some feat
...
+This applies to all unescaped identifiers in Cypher, such as label expressions, properties, variable names or parameters. In the given example, the quoted identifier would be \`my
identifier`.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
This applies to all unescaped identifiers in Cypher, such as label expressions, properties, variable names or parameters. 
In the given example, the quoted identifier would be \`myidentifier`.
```

**Review by AlexicaWright — COMMENTED** [review]
One sentence per line, but otherwise 👍 !

**Pull request merged**